corehq/apps/app_manager/suite_xml/xml_models.py

```diff
@@ -866,6 +866,8 @@ class Field(OrderedXmlObject):
 
     sort = StringField('@sort')
     print_id = StringField('@print-id')
+    lazy_loading = BooleanField('@lazy_loading', required=False)
+    cache_enabled = BooleanField('@cache_enabled', required=False)
```

Comment:
this one is the xml object corresponding to each column used when generating suite file.